ethstorage/miner/worker.go

```diff
@@ -371,21 +439,17 @@ func (w *worker) resultLoop() {
 			// optimistically check next result if exists
 			w.notifyResultLoop()
 		case <-ticker.C:
+			for shardId, s := range w.submissionStates {
+				log.Info(fmt.Sprintf("Mining stats since %s", startTime), "shard", shardId, "succeeded", s.Succeeded, "failed", s.Failed, "dropped", s.Dropped)
+			}
 			if len(errorCache) > 0 {
-				log.Error(fmt.Sprintf("Mining stats since %s", startTime),
-					"succeeded", succeeded,
-					"failed", len(errorCache),
-					"dropped", dropped,
-					"lastError", errorCache[len(errorCache)-1],
-				)
-			} else {
-				log.Info(fmt.Sprintf("Mining stats since %s", startTime),
-					"succeeded", succeeded,
-					"failed", len(errorCache),
-					"dropped", dropped,
-				)
+				log.Error(fmt.Sprintf("Mining stats since %s", startTime), "lastError", errorCache[len(errorCache)-1])
 			}
+			w.saveStates()
```

Comment:
Given that the mining data changes very slowly, can we configure a less frequent timer for state saving to optimize performance?